website/project/model.py

```diff
@@ -637,7 +637,7 @@ class Node(GuidStoredObject, AddonModelMixin):
 
     # Dictionary field mapping user id to a list of nodes in node.nodes which the user has subscriptions for
     # {<User.id>: [<Node._id>, <Node2._id>, ...] }
-    child_node_subscriptions = fields.DictionaryField()
+    child_node_subscriptions = fields.DictionaryField(default={})
```

Comment:
`default` can receive a callable. We should pass `dict` here instead of `{}` so that we get a new `dict` each time the field is initialized.